### ISSUE TITLE
Something went wrong with the resource group deployment. Ending script.

### ISSUE THREAD

**Issue by vararth** [Azure-Samples/azure-digital-twins-unreal-integration#16]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Create new Azure free subscription
> Follow instructions from this Github for setting up the Azure resources

### Any log messages given by the failure
> Log file is attached
[log.txt](https://github.com/Azure-Samples/azure-digital-twins-unreal-integration/files/10072950/log.txt)

### Expected/Desired behavior
> Successful Deployment

### OS and Version?
> Current Powershell version

### Versions
> Current repository version

**Comment by adamlash** [comment]
Hey There,
It looks like there are some quota issues related to one of the pieces of infrastructure in your deployment:

{"code":"Unauthorized","message":"{\r\n  \"Code\": \"Unauthorized\",\r\n  \"Message\": \"This region has quota of 0 instances for your subscription. Try selecting different region or SKU.\",\r\n  \"Target\": null,\r\n  \"Details\": [\r\n    {\r\n      \"Message\": \"This region has quota of 0 instances for your subscription. Try selecting different region or SKU.\"\r\n    },\r\n    {\r\n      \"Code\": \"Unauthorized\"\r\n    },\r\n    {\r\n      \"ErrorEntity\": {\r\n        \"ExtendedCode\": \"52039\",\r\n        \"MessageTemplate\": \"{0}. Try selecting different region or SKU.\",\r\n        \"Parameters\": [\r\n          \"This region has quota of 0 instances for your subscription\"\r\n        ],\r\n        \"Code\": \"Unauthorized\",\r\n        \"Message\": \"This region has quota of 0 instances for your subscription. Try selecting different region or SKU.\"\r\n      }\r\n    }\r\n  ],\r\n  \"Innererror\": null\r\n}"}]}}


Not sure what SKU/Piece of Infrastructure that is, but this is most probably the cause. Take a look at the deployment status under the resource group to get a deeper dive. Documentation here: https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history?tabs=azure-portal

**Comment by vararth** [comment]
@adamlash I am a beginner with azure, I have more experience with Unreal Engine.. could you please provide a simple solution/procedure for setting up the resource group in a bug-free manner? Since the only real-time Digital Twin webinar by Unreal has been using the WSP Office scene, would be nice if I could set it up without errors and bugs from services that I am not likely to dive deep into..

**Comment by adamlash** [comment]
@vararth there is a video here that may help with your setup: https://www.youtube.com/watch?v=TqUpTJhmxUc

Specific to the error you are getting, you will need to check the deployment like I have linked to see what service is hitting a quota limit.

Best Regards,
Adam Lash

**Comment by vararth** [comment]
@adamlash I was following the same webinar before I opened this GitHub page. The speaker/presenter does not cover the resource deployment on azure but instructs us to open the linked GitHub page and follow the directions - after doing, well we are here now.

Regardless, an update - I deleted all the content I have in my free azure subscription and started again - it worked this time but with a tiny issue:
I could not delete a particular "SignalR" type object created in the original resource group deployment. Unsure if it will cause any issues or not, please advise on how I can delete the relic.
![bug](https://user-images.githubusercontent.com/66017370/203697373-38222aa2-c41e-43ec-b1d5-2a4b8003ab87.png)

**Comment by adamlash** [comment]
It may be taking a while to delete or reflect in the portal, did this eventually delete?

**Comment by vararth** [comment]
@adamlash no, it is still not deleting.. 

Edit: The error I get when trying to delete it is this -

>ajaxExtended call failed 

Edit2: Tried restarting the service and tried deleting it again, below are the notifications I got -
>Restart of SignalR resource 'signalr-b62172b8' failed with status: 'error' and error:'Cannot restart resource in current state'

>Deletion of SignalR resource 'signalr-b62172b8' failed with status: 'error' and error: 'There is another running operation. You cannot delete the resource before it's completed.'

>Error details
>signalr-b62172b8: There is another running operation. You cannot delete the resource before it's completed. (Code: Conflict)

**Comment by Marco-Minchala** [comment]
Does anyone know why at the moment of pressing play in Unreal Engine the sensors are not displayed. Because according to all the things I've done everything is fine.

**Comment by adamlash** [comment]
> Does anyone know why at the moment of pressing play in Unreal Engine the sensors are not displayed. Because according to all the things I've done everything is fine.

Hey, when you say sensors are not displayed do you mean the telemetry, or the ability to select the sensors?

**Comment by Marco-Minchala** [comment]
@adamlash No, I meant that I make the connection with mock devices and also the configuration in unreal and in the end everything is fine, I didn't get any errors. So I repeated the whole process and in the end it worked but the temperature heatmap doesn't work and the hvca air particles don't change color, they are transparent. Maybe some help for it?

**Comment by vararth** [comment]
@Marco-Minchala yeah, I've had the same issue, temperature heatmap does not show up in other maps which I configure, nor do I get colours in the hvac air particles
